### PR TITLE
Remove spaghetii code 🍝

### PR DESCRIPTION
**Pull request by raluvy95**
Why not, it makes your life more easier~~

These changes makes into dynamic imports, which means you **don't need** to add extra line when new file is added into folder `cogs`.<br>
I hope it should be worked.<br> *Imagine trying to test on mobile*

### CODE REVIEW

File: bot/__main__.py [hunk 1]
```diff
@@ -31,17 +32,18 @@
...
+cogs = [
+    f[:-3] for f in listdir("bot/cogs") 
+    if f.endswith(".py")
+]
+moderation = [
+    f[:-3] for f in listdir("bot/cogs/moderation")
+    if f.endswith(".py")
+]
+
+for loadfiles in cogs:
```
Comment (AtieP) [inline bot/__main__.py]: I recommend you to rename `loadfiles` to `cog`.

Comment (raluvy95) [reply bot/__main__.py]: ah ok

File: bot/__main__.py [hunk 2]
```diff
@@ -31,17 +32,18 @@
...
+    bot.load_extension(f"bot.cogs.{loadfiles}")
+for files in moderation:
```
Comment (AtieP) [inline bot/__main__.py]: I recommend you to rename `files` to `cog`.

File: bot/__main__.py [hunk 3]
```diff
@@ -31,17 +32,18 @@
...
-
-bot.load_extension("bot.cogs.moderation.infractions")
-bot.load_extension("bot.cogs.moderation.member_log")
-bot.load_extension("bot.cogs.moderation.message_log")
-bot.load_extension("bot.cogs.moderation.silence")
+cogs = [
+    f[:-3] for f in listdir("bot/cogs") 
+    if f.endswith(".py")
+]
+moderation = [
```
Comment (AtieP) [inline bot/__main__.py]: I also recommend you to rename `moderation` to `moderation_cogs`.

**Review by AtieP — CHANGES_REQUESTED** [review]

**Review by AtieP — APPROVED** [review]

**Pull request merged**